src/__tests__/AddMovieModal.test.jsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import selectEvent from 'react-select-event';
+import AddMovieModal from '../components/modals/AddMovieModal/AddMovieModal';
+import configureStore from 'redux-mock-store';
+import {
+  MOVIES_INITIAL_STATE,
+  SEARCH_INITIAL_STATE,
+} from '../store/initial-state';
+import { mockMovies } from './test-mock-data';
+import { Provider } from 'react-redux';
+
+const stateInitial = {
+  moviesReducer: MOVIES_INITIAL_STATE,
+  searchReducer: SEARCH_INITIAL_STATE,
+};
+
+describe('Add/Edit Movie Modal', () => {
+  const mockStore = configureStore([]);
+  const mockAction = jest.fn();
+  let store;
+  beforeEach(() => {
+    store = mockStore(stateInitial);
+  });
+  test('AddMovie is shown', () => {
+    const { queryByText, queryByRole } = render(
+      <AddMovieModal show={true} onAction={mockAction} />,
+      {
+        wrapper: ({ children }) => (
+          <Provider store={store}>{children}</Provider>
+        ),
+      }
```

Comment:
I think that you can move it to separate function to reuse it. Like this:
```js
const getWrapper = () => ({
         wrapper: ({ children }) => (
          <Provider store={store}>{children}</Provider>
        ), 
})
```

Reply:
Ok, thanks